sawtooth-daml-common/src/main/java/com/blockchaintp/utils/LogUtils.java

```diff
@@ -0,0 +1,122 @@
+package com.blockchaintp.utils;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.core.config.Configurator;
+
+/**
+ * Utility functions that provide support for log4j functionality that is unavailable when using a facade
+ * like SLF4J or Apache Commons Logging.
+ */
+public final class LogUtils {
+
+  private static final Logger LOG = LogManager.getLogger();
+
+  private static final int WARN_LOG = 0;
+  private static final int INFO_LOG = 1;
+  private static final int DEBUG_LOG = 2;
+  private static final int TRACE_LOG = 3;
+
+  /**
+   * Sets the log4j log {@link Level level} for all loggers given a log level name. The log level name
+   * is not case sensitive, so, for example, "warn", "WARN" and "Warn" are all valid names for the log4j
+   * <code>WARN</code> log level.
+   *
+   * @param levelName the name of the new log4j log level for all loggers
+   */
+  public static void setRootLogLevel(final String levelName) {
+    Level level = Level.valueOf(levelName.toUpperCase());
```

Comment:
final

Reply:
fixed.  though technically speaking the style requirement (which are enforced in d-o-s) is that params be final and RO member vars be final, with no statement about the rest.